eventsources/eventing.go

```diff
@@ -435,7 +435,7 @@ func (e *EventSourceAdaptor) run(ctx context.Context, servers map[apicommon.Even
 				logger.Info("exiting eventbus connection daemon...")
 				return
 			case <-ticker.C:
-				if e.eventBusConn == nil || e.eventBusConn.IsClosed() {
+				if e.eventBusConn == nil || e.eventBusConn.IsInterfaceValueNil() || e.eventBusConn.IsClosed() {
```

Comment:
There is another way to do this that is more conventional, have the func perform a nil chek:

```golang
func (e *eventBusConn) IsClosed() {
  return e == nil || ...
}
````